src/components/inspector/chart-selector.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div>
+    <label class="typo__label">Select Chart Type</label>
+    <multiselect
+      v-model="selectedOption"
+      :options="options"
+      placeholder="Select an option"
+      label="title"
+      @input="selectOption"
+    ></multiselect>
```

Comment:
there is no value to render, so close the tag **/>** instead of  ></multiselect>

Reply:
solved.